service/src/test/resources/application-azure-test.yml

```diff
@@ -0,0 +1,20 @@
+workspace:
+  azure-test:
```

Comment:
I think I had to put:
```
workspace:
  crl:
    use-crl: true
    use-janitor: true

  azure-test:
    etc
```
to get Janitor to work when testing locally.

Reply:
All of the azure tests derive from BaseAzureTest. I made that enable both `azure-test` and `connected-test` profiles, so they get the CRL and Janitor setup from the connected test properties. Only the azure-specific things need to be in azure-test.